### PR TITLE
Remove event framework

### PR DESCRIPTION
**Pull request by tamalsaha**
This will greatly simplify code base and make it easier to fix bugs.

### CODE REVIEW

File: pkg/operator/certificate_tprs.go [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+		},
+	}
+	_, ctrl := cache.NewInformer(lw,
+		&sapi.Certificate{},
+		c.SyncPeriod,
+		cache.ResourceEventHandlerFuncs{
+			AddFunc: func(obj interface{}) {
+				if cert, ok := obj.(*sapi.Certificate); ok {
+					log.Infof("%s %s@%s added", cert.GroupVersionKind(), cert.Name, cert.Namespace)
+
```
Comment (sadlil) [inline pkg/operator/certificate_tprs.go]: cert controller not handling the event.

File: pkg/operator/endpoints.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+					log.Errorln(errors.New("Invalid Endpoints object"))
+					return
+				}
+
+				if !c.ServiceExists(newEndpoints) {
+					log.Warningf("Skipping Endpoints %s@%s, as it has no matching service", newEndpoints.Name, newEndpoints.Namespace)
+					return
+				}
+
+			},
```
Comment (sadlil) [inline pkg/operator/endpoints.go]: Needs to handle endpoint update with respect to ingress

File: pkg/operator/services.go [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+	return err
+}
+
+func (w *Operator) UpgradeAllEngress(service, providerName, cloudConfig string,
+	kubeClient clientset.Interface,
+	extClient acs.ExtensionInterface,
+	promClient pcm.MonitoringV1alpha1Interface,
+	store stash.Storage,
+	ingressClass string,
+	operatorServiceAccount string) error {
```
Comment (sadlil) [inline pkg/operator/services.go]: consider moving all those parameter to option

**Comment by tamalsaha** [comment]
RIP events framework!

**Pull request merged**